packages/cli/src/bin/index.ts

```diff
@@ -64,6 +64,7 @@ const errorHandler = (err: ClientError) => {
         .option("-L, --log", "Logs all API requests in detail", conf.log)
         .option("-a, --api-url <url>", "Specify base API url", conf.apiUrl)
         .option("-f, --format <value>", "Specify display formatting: json or pretty", conf.format)
+        .option("-f, --middlewareApiUrl <value>", "Specify middleware API url", conf.middlewareApiUrl)
```

Comment:
This should not be camel case.

Reply:
We should consider a new name for api-url too... With a fallback for backward compatibility, but preferably not documented